scripts/ukf.py

```diff
@@ -15,7 +15,7 @@
 # default UKFTractography parameters
 ukfdefaults = ['--numTensor', 2, '--stoppingFA', 0.15, '--seedingThreshold', 0.18, '--Qm', 0.001, '--Ql', 70,
                '--Rs', 0.015, '--stepLength', 0.3, '--recordLength', 1.7, '--stoppingThreshold', 0.1,
-               '--seedsPerVoxel', 10, '--recordTensors']
+               '--seedsPerVoxel', 1, '--recordTensors', '--numThreads', 8]
```

Comment:
Yogesh asked us to use just 1 `seedsPerVoxel`. At the same time, we specify fewer number of threads to facilitate multiprocessing.